crates/rpc-types-eth/src/transaction/request.rs

```diff
@@ -767,6 +766,34 @@ impl From<TxEnvelope> for TransactionRequest {
 #[non_exhaustive]
 pub struct TransactionInputError;
 
+/// Options for conditional raw transaction submissions.
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct ConditionalTxOptions {
```

Comment:
these should be moved to a separate module

Reply:
> these should be moved to a separate module

yeah you're totally right here boss, can you please suggest a module or i should create new file for this ?